### PR TITLE
chore(webpack): split prod/dev configs

### PR DESCRIPTION
**Pull request by davidholyko**
#### Summary

- Work Type: Chore 

#### What I did

- Split prod and dev
- Reduces Prod startup time. Before takes 20 seconds to load. Now ~4

#### Before / After

bundle at 12 MB
Screen Shot 2020-12-12 at 1 01 17 PM

bundle at 2 MB
Screen Shot 2020-12-12 at 1 02 28 PM

#### Checklist

- [x] Branch is in format `TYPE/scope/description`
- [x] PR title is in format `TYPE(scope): description`
- [x] Commits are in semantic format
- [x] Has Summary
- [x] Has Labels

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -3,108 +3,112 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
...
+    rules,
   },
+  plugins,
   devServer: {
     port: 3000,
     open: true,
     hot: true,
   },
   resolve: {
-    alias: {
```
Comment (PatrickDohn) [inline webpack.config.js:55]: Where did all this stuff go?

File: webpack.config.js [hunk 2]
```diff
@@ -3,108 +3,112 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 const CopyPlugin = require('copy-webpack-plugin');
 const webpack = require('webpack');
 
-module.exports = {
```
Comment (PatrickDohn) [inline webpack.config.js:6]: What happened to module.exports?

**Review by PatrickDohn — APPROVED** [review]
Just added some questions other than that LGTM

File: webpack.config.js (hunk 2 above)
Comment (davidholyko) [reply webpack.config.js:6]: moved lower in file.

https://github.com/starter-code/res-gen/blob/34e37edf51e2f8f219ebdc0a2f6fc64384288668/webpack.config.js#L90-L114

**Pull request merged**